io/io/v7/inc/ROOT/TFile.hxx

```diff
@@ -49,8 +49,7 @@ private:
 
   /// Serialize the object at address, using the object's TClass.
   //FIXME: what about `cl` "pointing" to a base class?
-  void WriteMemoryWithType(std::string_view name, const void *address,
-                           TClass *cl);
+  void WriteMemoryWithType(const std::string &name, const void *address, TClass *cl);
```

Comment:
Hi Axel,
out of curiosity -- and to educate myself -- why is a `const std::string&` fine here and not above? You foresee a different usage? I.e. never `WriteMemoryWithType("pooh")`?